### DISCUSSION TITLE
Is it possible to only read zip files and exclude other files, such as 7z?

### DISCUSSION THREAD

**Discussion by nyanpasunyan** [Difegue/LANraragi#713] [Q&A] [answered]
I am using the Windows version of lanraragi. Is there a way to let him read only certain files of the target folder and exclude other files, for example, only read zip files and exclude other files, such as 7z?

**Comment by Difegue** [comment, marked as answer]
~~no sort your files better oh my god what is wrong with you~~  

If you don't mind a bit of modding, you can open the LRR files in the Windows explorer, browse to this file: [/lib/LANraragi/Utils/Generic.pm](https://github.com/Difegue/LANraragi/blob/dev/lib/LANraragi/Utils/Generic.pm) and modify the `is_archive` function to remove the extensions you don't want the server to handle.  

Keep in mind those modifications will be removed when you update the app! 👍

**Reply by nyanpasunyan** [reply]
thank you for your help
